### PR TITLE
Chore/return published message reference

### PR DESCRIPTION
**Pull request by Bounteous17**
### Type:
* ~cicd: helm, docker & cicd adjustments.~
* feature: new capabilities.
* ~fix: bug, hotfix, etc.~
* ~refactor: enhacements.~
* other: docs, tests.

### What's the focus of this PR:
Since the possibility to publish scheduled messages is available. We also need now to be able to cancel scheduled messages that has not been consumed yet.

### Task list:

:cyclone: **Components**
- [x] `cancelScheduledMessages` new method available

:heavy_check_mark: **Test**
- [x] Canceled messages should not be consumed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -46,6 +46,11 @@ module.exports = () => {
 			return topicApi.publish(sender);
 		};
 
+		const cancelScheduledMessages = async (publicationId, sequenceNumber) => {
+			debug(`Cancelling messages ${sequenceNumber} for topic ${publicationId}`);
+			return sendersByPublication[publicationId].cancelScheduledMessages(sequenceNumber);
```
Comment (inigomarquinez) [inline index.js:51]: Should we check if `sendersByPublication[publicationId]` exists? We could potentially send a wrong `publicationId` so we would be calling `undefined.cancelScheduledMessages(sequenceNumber);`

**Review by inigomarquinez — COMMENTED** [review]
Nice! I added one question"

File: index.js (hunk 1 above)
Comment (Bounteous17) [reply index.js]: https://github.com/guidesmiths/systemic-azure-bus/pull/72/commits/8ccf8f1f091d8cea48e301e23bd59af935c4f8d5

**Review by inigomarquinez — APPROVED** [review]
LGTM!

**Comment by Betisman** [comment]
👍

**Pull request merged**